api/envoy/config/metrics/v3/stats.proto

```diff
@@ -220,7 +220,11 @@ message TagSpecifier {
   //
   // .. note::
   //
-  //   It is invalid to specify the same tag name twice in a config.
+  //   A stat name may be spelled in such a way that it matches two different
+  //   tag extractors for the same tag name. In that case, all but one of tag
+  //   values will be dropped. It is not specified which tag value will be
+  //   retained. The extraction will only occur for one of extractors, and only
```

Comment:
grammar: one of extractors

Reply:
thanks, caught & fixed another one also.